recipe/meta.yaml

```diff
@@ -24,6 +24,8 @@ outputs:
       build:
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - python                                 # [build_platform != target_platform]
```

Comment:
This looks wrong, @maresb ?

Reply:
Agreed. They are duplicated (as list items but not keys), so they are redundant. No big deal, we just remove them. And the other change is effectively nothing.